guides/customGuideAssets/ImprovedChooserGrid/js.js

```diff
@@ -0,0 +1,28 @@
+(function wireNextGuideButton(step) {
```

Comment:
A few thoughts on this one:

1) Maybe it's worth renaming this to "ChooserGridDynamicLinks"?  I think "Improved" is a bit too ambiguous here, since the most significant improvement added is the ability to add dynamic links.
2) The existing chooser grid snippet has a really clean way to add multiple event listeners simultaneously as you're adding new panels in the HTML.  Your HTML/CSS is arguably a bit more streamlined, but I'd really like to see the multi-eventListener attachment like you can see [here](https://github.com/pendo-io/snippets/blob/master/guides/customGuideAssets/chooser-grid/chooser.js).

Reply:
By adding the multi-attach, it's much easier for less JS-savvy users to implement a choose grid -- they'll probably be able to sub out CSS selectors where necessary, but I'm less confident they'd know how to structure event attachment functions

Reply:
changes implemented.

Reply:
reverted the name to ChooserGridDynamicLinks.

Reply:
for some reasone the changes I made to the code to have multi attach is not reflecting here.